.gitlab-ci.yml

```diff
@@ -553,6 +553,8 @@ puppy_deb-x64:
   variables:
     AWS_CONTAINER_CREDENTIALS_RELATIVE_URI: /credentials
     AGENT_MAJOR_VERSION: 7
+    # FIXME: we set PYTHON_RUNTIMES to '3' here so that `inv agent.omnibus-build` picks up the v7 tag
```

Comment:
👍 - I did vaguely recall this was probably an omnibus thing. Definitely a hack we need to fix, but good for now.

Reply:
yeah, it's again related to how we use env vars in the invoke tasks